extensions/wikia/AdEngine/js/video/player/porvata/porvataTracker.js

```diff
@@ -23,6 +23,18 @@ define('ext.wikia.adEngine.video.player.porvata.porvataTracker', [
 			'wikiaAdStop': 'closed'
 		};
 
+	function getContentType(player) {
+		if (!player) {
+			return;
+		}
+
+		var ad = player.ima.getAdsManager().getCurrentAd();
+
+		if (ad) {
+			return ad.getContentType();
+		}
+	}
```

Comment:
Maybe try to reduce the number of `return` for this function (if it's possible). I think it shouldn't affect to the execution time